docs/content/docs/tutorials/emr-on-eks-example.md

```diff
@@ -314,31 +331,26 @@ Simply run these commands to cleanup your environment
 ```
 # delete all custom resources
 kubectl delete -f virtualcluster.yaml
-# note: you cannot delete jobruns unless you want to cancel them
+kubectl delete -f jobrun.yaml
+# note: you cannot delete jobruns until virtualcluster its mapped to is deleted
 
 # uninstall emrcontainers controller
-helm delete ack-$SERVICE-controller -n $ACK_SYSTEM_NAMESPACE
+helm delete ack-$SERVICE-controller -n $ACK_K8S_NAMESPACE
 
 # delete namespace
 kubectl delete ns $ACK_K8S_NAMESPACE
 kubectl delete ns $EMR_NAMESPACE
 
+# delete aws resources
+aws logs delete-log-group --log-group-name=/emr-on-eks-logs/$EKS_CLUSTER_NAME
+aws s3 rm s3://$EKS_CLUSTER_NAME-$RANDOM_ID1 --recursive
+aws s3 rb s3://$EKS_CLUSTER_NAME-$RANDOM_ID1 
+
 # delete EKS cluster
 eksctl delete cluster --name "${EKS_CLUSTER_NAME}"
 ```
 ## Limitations
 
-* JobRun CRD doesn’t have complete features available due to an issue with how ACK handles cyclic structure. Until this [issue](https://github.com/aws-controllers-k8s/community/issues/1445) is resolved, you can’t specify configurations from **`configuration overrides`** structure.
-```
-# configurationOverrides:
-#   applicationConfiguration:
-#   monitoringConfiguration:
-#     cloudWatchMonitoringConfiguration:
-#       logGroupName:
-#       logStreamNamePrefix:
-#     s3MonitoringConfiguration:
-#       logUri:
-```
 * You cannot delete a JobRun unless its in **error** state. There is no delete-job-run API for deleting jobs (for good reason). However, if your JobRun goes into error state, you can run `kubectl delete jobrun/<job-run-name>` to cancel the job.
```

Comment:
What are you planning to do with this? How does the controller reacts to JobRun deletions?

Reply:
emr-containers doesn't have delete jobrun api. I'm going to create separate issue but want to hear from customers first if this is an issue for them